telemetry/metrics.go

```diff
@@ -1,10 +1,31 @@
 package telemetry
 
 import (
+	"sort"
+	"strconv"
+	"strings"
+
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
+var DefaultHistogramBuckets = []float64{
+	0.005, // 5 ms
+	0.01,  // 10 ms
+	0.025, // 25 ms
+	0.05,  // 50 ms
+	0.1,   // 100 ms
+	0.25,  // 250 ms
+	0.5,   // 500 ms
+	1,     // 1 s
+	2.5,   // 2.5 s
+	5,     // 5 s
+	10,    // 10 s
+	30,    // 30 s
+	60,    // 60 s
+	300,   // 5 min
+}
```

Comment:
Reduce the default total number of buckets to these:
```suggestion
var DefaultHistogramBuckets = []float64{
	0.05,  // 50 ms
	0.5, // 500 ms
	5,     // 5 s
	30,    // 30 s
}
```